src/HttpLambda.ts

```diff
@@ -0,0 +1,25 @@
+import { Handler, APIGatewayProxyEventV2 } from 'aws-lambda';
+import { IHttpLambda } from './IHttpLambda';
+import { IServiceContainer, Newable } from "@aesop-fables/containr";
+import { IHttpEndpoint } from './IHttpEndpoint';
+
+export class HttpLambda implements IHttpLambda{
```

Comment:
I think I understand the thought process behind why you made this into a class. However, it mights most JS norms in terms of the API we expose to folks. 

I think we want to be able to do:

```ts
export const handler = createHttpLambda(...
```

right now you'd force them to do (for every file and endpoint they want to create):
```ts
const lambda = new HttpLambda();
export const handler = lambda.createHttpLambda(...
```